core/app/[locale]/(default)/webpages/[id]/contact/_actions/submit-contact-form.ts

```diff
@@ -96,18 +96,9 @@ export async function submitContactForm<F extends Field>(
     // eslint-disable-next-line no-console
     console.error(error);
 
-    if (error instanceof BigCommerceGQLError) {
+    if (error instanceof BigCommerceAPIError) {
```

Comment:
Is there no way to get a GQL error from this action?

Reply:
@jorgemoya Ah, looks like this portion might have been a rebasing / conflict error on my part after your PR to update the GQL errors. I'll fix this